src/flask/config.py

```diff
@@ -107,7 +107,7 @@ def from_pyfile(self, filename: str, silent: bool = False) -> bool:
                          root path.
         :param silent: set to ``True`` if you want silent failure for missing
                        files.
-
+        :return: bool.``True`` if able to update values in config, ``False`` otherwise.
```

Comment:
I think the read the docs CI build requires a blank line after this one to pass.

Reply:
Also, remove the leading "bool.", and the trailing ", ``False`` otherwise``, they're both redundant.

Reply:
As in `from_pyfile`?

Reply:
"True if values were loaded successfully." sounds right for all the methods.

Reply:
Sure will update it